shell/browser/ui/views/win_caption_button.cc

```diff
@@ -52,7 +63,8 @@ void WinCaptionButton::OnPaintBackground(gfx::Canvas* canvas) {
   gfx::Rect bounds = GetContentsBounds();
   bounds.Inset(gfx::Insets::TLBR(0, 0, 0, 0));
```

Comment:
I manually applied almost all the changes from the upstream Electron PR, as well as all changes from the commit https://chromium.googlesource.com/chromium/src/+/30a32e6d6909e8e6e267b48bb7294e739b210c51%5E%21/#F0.
I left this line unchanged due to https://github.com/electron/electron/issues/35312.

Reply:
Please keep the changes as the original PR, so that any regressions further addressed in `main` can be backported.